src/content/security/index.md

```diff
@@ -0,0 +1,268 @@
+
+Watch more on the 2FA:
+
+<iframe width="100%" height="315px" src="https://www.youtube.com/embed/m8jlnZuV1i4?start=3479&end=3875" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Uninstall browser extentions {#uninstall-browser-extentions}
+
+Browser extensions like Chrome extensions or Add-ons for Firefox can augment useful browser functionality and improve user experience, but they come with risks. By default, most browser extensions ask for access to 'read and change site data', allowing them to do almost anything with your data. Chrome extensions are always automatically updated, so a previously safe extension may update later to include malicious code. Most browser extensions are not trying to steal your data, but you should be aware that they can.
+
+#### Stay safe by: {#browser-extention-safety}
+
+- Only install browser extensions from trusted sources
+- Removing unused browser extensions
+- Install Chrome extensions locally to stop auto-updating (Advanced)
+
+[More on the risks of browser extentions](https://www.kaspersky.co.uk/blog/browser-extensions-security/12750/)
+
+<Divider />
+
+## Crypto Security 101 {#crypto-security}
+
+### Upgrade your knowledge {#upgrade-your-knowledge}
+
+One of the biggest reasons people get scammed in crypto generally is a lack of understanding. For example, if you don't understand that the Ethereum network is decentralized and owned by no one, then it's easy to fall prey to someone pretending to be a customer service agent that promises to return your lost ETH in exchange for your private keys. Educating yourself on how Ethereum works is a worthwhile investment.
+
+<DocLink to="/what-is-ethereum/" title="What is Ethereum?" />
+
+<DocLink to="/eth/" title="What is ether?" />
+<Divider />
+
+## Wallet Security {#wallet-security}
+
+### Don't give out your private keys {#protect-private-keys}
+
+**Never, for any reason, share your private keys!**
+
+The private key to your wallet acts as a password to your Ethereum wallet. It is the only thing stopping someone who knows your wallet address from draining your account of all of its assets!
+
+<DocLink to="/wallets/" title="What's an Ethereum wallet?" />
+
+#### Don't take screenshots of your seed phrases/private keys {#screenshot-private-keys}
+
+By screenshotting your seed phrases or private keys, you risk syncing them to the cloud and potentially making them accessible to hackers. Obtaining private keys from the cloud is a common attack vector for hackers.
+
+### Use a hardware wallet {#use-hardware-wallet}
+
+A hardware wallet provides offline storage for private keys. They are considered the most secure wallet option for storing your private keys.
+
+Keeping private keys offline massively reduces the risk of being hacked, even if a hacker gets control of your computer.
+
```

Comment:
Should we mention mnemonic phrases here? Pretty much all hardware wallets (and most software wallets) utilize BIP-39 now, and this random list of words may be very confusing to those new to the space. 

People may not understand the differences/similarities between private keys and a mnemonic, and perhaps a quick note to remind people that these words should be the most guarded data you have in the crypto space. Also, guarding the mnemonic long-term involves more than theft protection, but also protection from natural disasters and damage such as fires/floods/crush/electrical shock (ie. put these words on steel).